autoload/neomake.vim

```diff
@@ -179,7 +181,13 @@ function! neomake#Make(options, ...) abort
     let file_mode = get(a:options, 'file_mode')
 
     let enabled_makers = get(a:options, 'enabled_makers', [])
-    if !len(enabled_makers)
+    let sh_command = get(a:options, 'sh_command', '')
+    if len(sh_command)
```

Comment:
Instead of having a `sh_command` option, I think we should move the stuff inside this block into `s:NeomakeSh` in the other file.